packages/azure-themes/src/azure/AzureThemeDark.ts

```diff
@@ -80,6 +106,13 @@ const darkExtendedSemanticColors: Partial<IExtendedSemanticColors> = {
   statusWarningBackground: DarkSemanticColors.statusBar.warning,
   statusWarningText: DarkSemanticColors.text.body,
   statusWarningIcon: CommonSemanticColors.icons.warning,
+  textFieldBorderDisabled: DarkSemanticColors.primaryButton.disabled.border,
+
+  // temporary work around for high contrast themes
```

Comment:
Are you planning to fix this temporary workaround? If so, please create a workitem and mention the workitem number here.